rxjava-modules/rxjava-observables/src/test/java/com/baeldung/rxjava/combine/ObservableMergeVsConcatUnitTest.java

```diff
@@ -0,0 +1,127 @@
+package com.baeldung.rxjava.combine;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import org.junit.Test;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import rx.Observable;
+import rx.observers.TestSubscriber;
+
+public class ObservableMergeVsConcatUnitTest {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ObservableMergeVsConcatUnitTest.class);
```

Comment:
According to https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names this should be lowercase since is is not actually deeply immutable
```suggestion
    private static final Logger logger = LoggerFactory.getLogger(ObservableMergeVsConcatUnitTest.class);
```